SCF1/item.java

```diff
@@ -0,0 +1,219 @@
+package assignment1;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class item 
+	{
+	String itemname[] = { "Biscuits ", "Drinks    ", "Chips     ","Fruits   ","Vegiees   " };
+	int price[] = { 10, 20, 30, 20, 30 };
+	HashMap<String, Integer> map = new HashMap<>();
+	
+	// To show the available item list
+	public void show() 
+		{
```

Comment:
Read about code formatting in java.  
for example : 
```
public void show() {
     //code goes here.
}
```
same for the case of if and for statements